packages/cpp-debug/src/browser/cpp-memory-widget.tsx

```diff
@@ -36,7 +36,7 @@ export class MemoryView extends ReactWidget {
     protected startAddress: number = 0;
     protected bytes: Uint8Array | undefined = undefined;
     // If bytes is undefined, this string explains why.
-    protected memoryReadError: string = 'Nothing to see here.';
+    protected memoryReadError: string = 'No memory address information currently available.';
```

Comment:
I wouldn't say 'No memory address information currently available.' (it's the "memory address information" part that I think sounds weird), but more "No memory contents currently available".

Reply:
Sure I'll make the adjustments, I was just trying to find something more meaningful than
`Nothing to see here` 😄